app/controllers/ops_controller/db.rb

```diff
@@ -26,7 +26,7 @@ def db_list(exp = nil)
         ]
       )
     elsif model == VmdbDatabaseConnection
-      @zones = Zone.all.sort_by(&:name).collect { |z| [z.name, z.name] }
+      @zones = Zone.visible.all.sort_by(&:name).collect { |z| [z.name, z.name] }
```

Comment:
2 things

- The `.all` is no longer needed.
- Is it better to do the sorting in SQL with `.order(:name)`?